src/cryptoadvance/specter/devices/hwi/jade.py

```diff
@@ -151,104 +178,296 @@ def get_master_blinding_key(self) -> str:
     def sign_tx(self, tx: PSBT) -> PSBT:
         """
         Sign a transaction with the Blockstream Jade.
-
-        - Jade can only be used to sign single-key inputs at this time.   It cannot sign multisig or arbitrary scripts.
         """
+        # Helper to get multisig record for change output
+        def _parse_signers(
+            hd_keypath_origins: List[KeyOriginInfo],
+        ) -> Tuple[List[Tuple[bytes, Sequence[int]]], List[Sequence[int]]]:
```

Comment:
This looks very frightening to me. Shouldn't we maybe create classes to that data-structure?

Reply:
It is a copy-paste from HWI, so I would keep it as it is there - then we don't diverge from HWI
https://github.com/bitcoin-core/HWI/blob/master/hwilib/devices/jade.py#L172